.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
What is this used for?

Reply:
This is to prevent windows changing `\n` into `\r\n` its why [a4999e6](https://github.com/linz/basemaps/pull/2880/commits/a4999e65e8d19e10ca62fd3d345f2dfdc9f4889e) failed to build.